docs/self-hosted/config-options.md

```diff
@@ -346,7 +346,7 @@ Redis is required when you run Directus load balanced across multiple containers
 | `IP_CUSTOM_HEADER`               | What custom request header to use for the IP address                                                                                                                                                 | false                     |
 | `ASSETS_CONTENT_SECURITY_POLICY` | Custom overrides for the Content-Security-Policy header for the /assets endpoint. See [helmet's documentation on `helmet.contentSecurityPolicy()`](https://helmetjs.github.io) for more information. | --                        |
 | `IMPORT_IP_DENY_LIST`            | Deny importing files from these IP addresses. Use `0.0.0.0` for any local IP address                                                                                                                 | `0.0.0.0,169.254.169.254` |
-| `CONTENT_SECURITY_POLICY_*`      | Custom overrides for the Content-Security-Policy header. See [helmet's documentation on `helmet.contentSecurityPolicy()`](https://helmetjs.github.io) for more information.                          | --                        |
+| `CONTENT_SECURITY_POLICY_*`      | Custom overrides for the Content-Security-Policy header. See [helmet's documentation on `helmet.contentSecurityPolicy()`](https://helmetjs.github.io) for more information. E.g. `CONTENT_SECURITY_POLICY_DIRECTIVES__CONNECT_SRC="self http://example.com"`                         | --                        |
```

Comment:
This is getting too long IMO. If we really want to add an example, we should place that below the table, like we're already doing it for other config options.